android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -1169,6 +1173,60 @@ class JsonConverter {
             playerViewConfig = toPlayerViewConfig(json),
             pictureInPictureConfig = toPictureInPictureConfig(json.getMap("pictureInPictureConfig")),
         )
+
+
+        @JvmStatic
+        fun fromBufferLevel(bufferLevel: BufferLevel?): WritableMap? {
+            if (bufferLevel == null) {
+                return null
+            }
```

Comment:
```suggestion
```

Reply:
Address in 245d413e0f81d2455d76205f22b400731cc7c551  for Android
Address in a0b00ce2ef28efaa316700049bcb3b3671b464d7 for iOS